integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1304,3 +1304,30 @@ func (s *DockerSuite) TestRunUserDeviceAllowed(c *check.C) {
 	out, _ := dockerCmd(c, "run", "--device", "/dev/snd/timer:w", "busybox", "cat", file)
 	c.Assert(out, checker.Contains, fmt.Sprintf("c %d:%d w", stat.Rdev/256, stat.Rdev%256))
 }
+
+func (s *DockerDaemonSuite) TestRunWithSeccompJSONFormatV1(c *check.C) {
+	testRequires(c, SameHostDaemon, seccompEnabled)
+
+	err := s.d.StartWithBusybox()
+	c.Assert(err, check.IsNil)
+
+	jsonData := `{
+	"version": 1,
+	"defaultAction": "SCMP_ACT_ALLOW",
+	"syscalls": [
+		{
+			"names": ["chmod"],
+			"action": "SCMP_ACT_ERRNO"
```

Comment:
Can you add `fchmod` and `fchmodat` as per https://github.com/docker/docker/pull/24299 to this test